devices/opi-pc-plus/kali-rolling-armhf/README.md

```diff
@@ -0,0 +1,4 @@
+Kali Linux is a security-focused Debian-based distribution with pre-installed tools for penetration testing.
+It is maintained by Offensive Security Ltd. `opi-pc-plus/kali-rolling-armhf` allows building custom 32-bit Kali Linux images for [OrangePi PC Plus](http://orangepi.org/orangepipcplus/).
+## What's in the distribution?
+Kali Linux has the rolling release cycle.
```

Comment:
Kali Linux is a rolling release distribution, so it's pointless to list the packages with their versions here since the versions may change every day.